src/Test/AbstractConfigTestCase.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Test;
+
+use PhpCsFixer\ConfigInterface;
+use PhpCsFixer\Fixer\DeprecatedFixerInterface;
+use PhpCsFixer\Fixer\FixerInterface;
+use PhpCsFixer\FixerFactory;
+use PhpCsFixer\RuleSet;
+use PHPUnit\Framework\TestCase;
+
+abstract class AbstractConfigTestCase extends TestCase
+{
+    final protected function doTestAllDefaultRulesAreSpecified(ConfigInterface $config)
```

Comment:
what do you mean by `specified` ?

Reply:
I mean that if you upgrade PHP-CS-Fixer from [2.14](https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/tag/v2.14.0) to [2.15](https://github.com/FriendsOfPHP/PHP-CS-Fixer/releases/tag/v2.15.0) without updating your `.php_cs`, the test will fail because the new [`php_unit_size_class`](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/3939) rule isn't configured: to pass the test you need to set it either to false or to whatever the rule allows you.

Maybe I should change the name of the function to `doTestAllDefaultRulesAreConfigured`, shouldn't I?

Reply:
`doTestAllBuiltInRulesAreConfigured`?

Reply:
I like it, done :+1: